pkg/network/event_common.go

```diff
@@ -302,6 +302,17 @@ func (c ConnectionStats) IsExpired(now uint64, timeout uint64) bool {
 	return c.LastUpdateEpoch+timeout <= now
 }
 
+// IsEmpty returns whether the connection has any statistics
+func (c ConnectionStats) IsEmpty() bool {
+	// TODO why does this not include TCPEstablished and TCPClosed?
```

Comment:
I have no objection with adding those fields here, do you @hmahmood?

Reply:
happy to do that in a different PR though

Reply:
actually we may set the closed flags on otherwise totally empty connections in kernelspace... so this could end up returning false a lot more

Reply:
This function was mostly checking for whether the connection exchanged any data, and almost exclusively for code that is storing closed connections in `state.go`. As opposed to something like `IsZero` which does check for those other two fields.